stock_inventory_discrepancy/models/stock_inventory.py

```diff
@@ -70,4 +78,5 @@ def action_done(self):
 
     @api.multi
     def action_force_done(self):
+        """Function to super action_force_done"""
```

Comment:
IMHO these lines are not very useful, it is good to add comments on the code but always to add some value, you are just saying "what" is being done which is obvious looking at the code, for me comments should be about "why" or any extra info that cannot be obtained reading the code itself.